weblab_cg/templates/chaste/normal_model.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef {% filter upper %}{{class_name}}FROMCELLML_HPP_{% endfilter %}
+#define {% filter upper %}{{class_name}}FROMCELLML_HPP_{% endfilter %}
+
+//! @file
+//! 
+//! This source file was generated from CellML.
+//! 
+//! Model: {{model_name_from_file}}
+//! 
+//! Processed by webalab_cg - CellML Tools in Python
+//!     (translators: , webalab_cg: , options: normal)
+//! on {{generation_date}}
+//! 
+//! <autogenerated>
```

Comment:
What's this? I'm feeling uneasy about the lack of a closing tag

Reply:
Dood point. That's what chaste generated code looked like (no closing tag. Easy enough to add one though.

Reply:
I'd leave things like that in comments alone, it probably excludes the file from doxygen coverage testing or something in Chaste tests.

Reply:
In general can we focus on comments that need addressing for stuff to work 😉 there is a cost to everything people get emails about and need to read and think about!

Reply:
sure